packages/grafana-ui/src/utils/valueFormats/categories.ts

```diff
@@ -54,6 +54,8 @@ export const getCategories = (): ValueFormatCategory[] => [
       { name: 'Degrees (°)', id: 'degree', fn: toFixedUnit('°') },
       { name: 'Radians', id: 'radian', fn: toFixedUnit('rad') },
       { name: 'Gradian', id: 'grad', fn: toFixedUnit('grad') },
+      { name: 'Arc-minutes', id: 'arcmin', fn: toFixedUnit('arcmin') },
```

Comment:
Shouldn't units be named `Arc Minutes` and `Arc Seconds`? 

Referring to https://en.wikipedia.org/wiki/Minute_and_second_of_arc

Reply:
Yes, should be better.